website/docs/guides/getting_started.html.markdown

```diff
@@ -8,54 +8,21 @@ description: |-
 
 # Getting Started with the New Relic Provider
```

Comment:
The markdown in this file is not rendering in the Github rich diff formatter, is that a clue that there is a problem with syntax somewhere inside?

Reply:
It might be the header at teh top `---` that is keeping it from rendering properly.

Reply:
I'll run the website locally and double check.

Reply:
Running locally makes the doc look good anyway.